### PR TITLE
update TensorFlow version

### PR DESCRIPTION
**Pull request by lm-jira**
## Motivation and Context
Because the latest subversion of TensorFlow. version 1 is 1.13 but Blueoil uses TensorFlow 1.4.1, we need to update TensorFlow version to the latest one.
After updating, we need to confirm that Blueoil works properly without error.
We also need to compare the performance of Blueoil before and after updating to see if the changing in TensorFlow affects Blueoil or not.

## Description

I changed the requirement and the docker files using TensorFlow from version 1.4.1 to 1.13.0.
I also changed cuda from version 8 to 10, and cudnn from version 6 to 7 to support TensorFlow 1.13.0.
Additional dependencies from updating  cuda version are added to `docker/Dockerfile` too.
@nlpng also created a LeakyRelu operators in the Blueoil.

## How has this been tested?
The detail of the test can be found in this [document](https://docs.google.com/document/d/1uN6nnZJOSiHHTKvH39Vk0AVhxS64cVITSncAn_NP9fQ/edit?usp=sharing).
Blueoil passes the following test without error and problem.
1. docker building
2. training model both from scratch and from checkpoint
3. convert model to library files
4. predict the image

The performance between before and after update the TensorFlow is comparable.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by ruimashita** [comment]
@lm-jira 
Plz refer to https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/dockerfiles/dockerfiles/devel-gpu.Dockerfile

**Comment by kchygoe** [comment]
rerun test

**Comment by kchygoe** [comment]
run test

**Comment by lm-jira** [comment]
run test

**Comment by iizukak** [comment]
run test

**Comment by iizukak** [comment]
@lm-jira Please resolve conflicts.

**Comment by iizukak** [comment]
@lm-jira This PR is still WIP ?

**Comment by lm-jira** [comment]
@iizukak It's already finished. Sorry for not updating it.

**Comment by ruimashita** [comment]
@lm-jira 
Why use `tf.Session()` instead of `tf.InteractiveSession()` ?

**Comment by lm-jira** [comment]
@ruimashita Sorry. That's just my mistake. 
I will edit it back to `interactivesession`.

**Comment by lm-jira** [comment]
It seems like using `with tf.InteractiveSession()` results in an attribute error.
I'm gonna change it back to create and close session manually with `sess = tf.InteractiveSession()` and `sess.close()`.

**Comment by ruimashita** [comment]
FYI
tensorflow TestCase class
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/framework/test_util.py#L1576

**Comment by lm-jira** [comment]
@ruimashita 
[lmnet] jenkin test raises this kind of warning so I want to close the interactive session after using it.
It is a warning so maybe it's better to merge this branch today and solve this warning in new PR.
```
lmnet/tests/lmnet_tests/test_layers.py::test_unpool
  /home/blueoil/lmnet/.tox/py36/lib/python3.6/site-packages/tensorflow/python/client/session.py:1702: UserWarning: An interactive session is already active. This can cause out-of-memory errors in some cases. You must explicitly call `InteractiveSession.close()` to release resources held by the other session(s).
    warnings.warn('An interactive session is already active. This can '
```

**Comment by lm-jira** [comment]
run test

**Comment by iizukak** [comment]
@ruimashita @nlpng Review Please 🙇

**Comment by ruimashita** [comment]
@lm-jira 

> [lmnet] jenkin test raises this kind of warning so I want to close the interactive session after using it.
It is a warning so maybe it's better to merge this branch today and solve this warning in new PR.

Sounds good :smile:

File: lmnet/tests/lmnet_tests/networks_tests/object_detection_tests/test_yolo_v1.py [hunk 1]
```diff
@@ -231,7 +231,6 @@ def test_convert_boxes_space_inverse():
                 [
                     [50, 100, 400, 200, ],
                 ],
-
```
Comment (ruimashita) [inline lmnet/tests/lmnet_tests/networks_tests/object_detection_tests/test_yolo_v1.py]: Please avoid unnecessary file change :-1:

File: lmnet/tests/lmnet_tests/networks_tests/object_detection_tests/test_yolo_v2.py [hunk 2]
```diff
@@ -687,7 +687,6 @@ def test_reorg():
     outputs = model._reorg("reorg", inputs, stride=2, data_format="NHWC", use_space_to_depth=False)
 
     tf.InteractiveSession()
-
```
Comment (ruimashita) [inline lmnet/tests/lmnet_tests/networks_tests/object_detection_tests/test_yolo_v2.py]: Please avoid unnecessary file change :-1:

**Comment by ruimashita** [comment]
@lm-jira @iizukak 
I think the google docs of test should be linked in this PR description.

File: lmnet/tests/lmnet_tests/networks_tests/object_detection_tests/test_yolo_v1.py (hunk 1 above)
Comment (lm-jira) [reply lmnet/tests/lmnet_tests/networks_tests/object_detection_tests/test_yolo_v1.py]: fixed it

File: lmnet/tests/lmnet_tests/networks_tests/object_detection_tests/test_yolo_v2.py (hunk 2 above)
Comment (lm-jira) [reply lmnet/tests/lmnet_tests/networks_tests/object_detection_tests/test_yolo_v2.py]: fixed it

**Comment by lm-jira** [comment]
@ruimashita 
I remove the unnecessary file change and update the description by adding the link to the test document.

**Pull request merged**